azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -38,6 +38,35 @@ def load_team_help():
     short-summary: Manage security groups
     """
 
+    helps['devops security group create'] = """
+    type: command
+    short-summary: Create a new Azure DevOps group or materialize an existing AAD group.
+    long-summary:
+    examples:
+          - name: Create an Azure DevOps Group with name and description
+            text: |
+              az devops security group create --name 'Some group name' --description
+              'Something to describe this group' 
+              
+          - name: Materialize an existing AAD Group with its origin ID/AAD object ID
+            text: |
+              Get object ID of the existing AAD group 
+              az ad group show -g {Group Name}
+              az devops security group create --origin-id {Object ID}
+              Use this descriptor to add this group to other Azure DevOps Groups.
+
+          - name: Materialize an existing AAD Group with its origin ID/AAD object ID and also add it to an AzureDevOps group.
+            text: |
+              Get object ID of the existing AAD group               
+              az ad group show -g {Group Name}
+              az devops security group create --origin-id {Object ID} --groups 'vssgp.someDescriptorForGroup'
```

Comment:
Have an example on bulk add as well - adding it to groups at the same time - what is the accepted method.